app/Http/Controllers/AirfieldController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Airfield;
+use App\Services\AirfieldService;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class AirfieldController extends BaseController
+{
+    /**
+     * @var AirfieldService
+     */
+    private $airfieldService;
+
+    /**
+     * AirfieldController constructor.
+     * @param AirfieldService $airfieldService
+     */
+    public function __construct(AirfieldService $airfieldService)
+    {
+        $this->airfieldService = $airfieldService;
+    }
+
+    /**
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function getAllAirfields(Request $request) : JsonResponse
+    {
+        $airfields = $request->get('controllers') === '1'
```

Comment:
Is this a boolean or a string? Could it be worth casting this to boolean?

Reply:
Originally intended to come in as a string in the URL like `https://ukcp.vatsim.uk/airfield?controllers=1`